### PR TITLE
test case for invoice service

### PR DESCRIPTION
**Pull request by ghost**
## Related Tickets
[- ticket redmine](https://edu-redmine.sun-asterisk.vn/issues/76202?issue_count=30&issue_position=1&next_issue_id=76192)

## WHAT (optional)
- Change number items `completed/total` in admin page.

## HOW
- I edit js file, inject not_vary_normal items in calculate function.

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)
image

### CODE REVIEW

**Comment by ghost** [comment]
ready

File: src/test/auth.test.ts [hunk 1]
```diff
@@ -20,8 +20,8 @@ describe('Register', () => {
   it('should return user if register successfully', async () => {
     const userDTO: UserRegisterDto = {
       username: 'Tran Trong Tin',
-      email: 'test11@gmail.com',
-      password: 'Test123@!!123',
+      email: 'test111@gmail.com',
+      password: 'Test1213@!!123',
     };
```
Comment (huongnt-2545) [inline src/test/auth.test.ts]: phần thay đổi code này hình như pull trc cũng có rồi mà nhể?

File: src/test/blog.test.ts [hunk 2]
```diff
@@ -0,0 +1,82 @@
+import { AppDataSource } from '../config/ormconfig';
```
Comment (huongnt-2545) [inline src/test/blog.test.ts]: rebase code mới nhất đi e nha

**Comment by ghost** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
merged

**Pull request merged**